packages/config/index.js

```diff
@@ -115,9 +115,10 @@ class ConfigManager extends EventEmitter {
           stringify: (data) => JSON5.stringify(data, null, 2)
         }
       case '.toml':
+      case '.tml':
```

Comment:
I don't think there is a matching test for the `tml` extension.

Reply:
for auto load there is `.tml` in file types to be tested: https://github.com/platformatic/platformatic/blob/feat/load-toml-and-yaml-file-as-fallback/packages/db/test/cli/auto-config.test.js#L10